.circleci/config.yml

```diff
@@ -189,7 +193,27 @@ jobs:
       - checkout
       - *step_process_codebase
       - *step_setup_remote_docker
-      - run: docker-compose build cli
+      - run:
+          name: Build site
+          command: ahoy build
```

Comment:
changed docker build, which did not run a site install (only was useful for dependencies check and Docker build check), with a proper build to make sure that a specific variation of the build finishes successfully.